website/docs/installation/configuration.mdx

```diff
@@ -144,19 +144,19 @@ These settings affect where media files are stored. Those files include applicat
 -   `AUTHENTIK_STORAGE__MEDIA__S3__SECURITY_TOKEN`: Security token to authenticate to S3. May be omitted. Supports hot-reloading.
 -   `AUTHENTIK_STORAGE__MEDIA__S3__BUCKET_NAME`: Name of the bucket to use to store files.
 -   `AUTHENTIK_STORAGE__MEDIA__S3__CUSTOM_DOMAIN`: Domain to use to create URLs for users. Mainly useful for non-AWS providers. May include a port. Must include the bucket. Example: `s3.company:8080/authentik-media`.
--   `AUTHENTIK_STORAGE__MEDIA__S3__SECURE_URLS`: Whether URLS created for users use `http` or `https`. Defaults to `true`.
+-   `AUTHENTIK_STORAGE__MEDIA__S3__SECURE_URLS`: Whether URLS created for users use `https` rather than `http`. Defaults to `true`.
 
 ## authentik Settings
 
 ### `AUTHENTIK_SECRET_KEY`
 
-Secret key used for cookie signing and unique user IDs, don't change this after the first install.
+Secret key used for cookie signing and unique user IDs; don't change this after the first install.
```

Comment:
Nitpick: This isn't consistent with the language on line 157.  

"Secret key used for cookie signing and unique user IDs. Do not change this after the first install."